Updating README-sycl.md to capture the 3.5 modifications (#16)
codeplaysoftware/cutlass-fork: README-sycl.md

Currently, only one example works on NVIDIA SM 80.

## Requirements
 

> To build CUTLASS SYCL support you need the latest version of DPC++ compiler, you can either use a recent [nighly build](https://github.com/intel/llvm/releases)

Reviewer: ```suggestion
To build CUTLASS with SYCL support, install the latest DPC++ compiler with the CUDA backend enabled, as described [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#build-dpc-toolchain-with-support-for-nvidia-cuda), and make sure to update the following environment path as follows - 

`export PATH=/path/to/llvm/build/install/bin:$PATH`
`export LD_LIBRARY_PATH=/path/to/llvm/build/install/lib:$LD_LIBRARY_PATH`
```

or build the compiler from source.

> In either case, make sure to enable the NVIDIA plugin so you can build applications

Reviewer: I do not suppose the Nvidia / AMD plugins can be installed unless the base toolkit is installed.
For this we need to build dpcpp from source with the Cuda backend enabled

Author: No you dont need to you can the plugins from codpelay nightly build.

Reviewer: I see, in that case let's not call it plugins, which confuses with the Plugins available on the codeplay's website to people who are completely new to SYCL,

we can phrase it as - 
```suggestion
To build CUTLASS with SYCL support, install the latest DPC++ compiler with the CUDA backend enabled, either by building from source as described [here](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#build-dpc-toolchain-with-support-for-nvidia-cuda) ,  or by downloading the [nightly releases](https://github.com/intel/llvm/releases)
```

# Running the example

## CuTe 
Currently, you can build the CuTe Tutorial using the following command: 

```
ninja [EXAMPLE_NAME]_sycl
```

You can run it like this from your build directory

```
LD_LIBRARY_PATH=/path/to/sycl/install/lib ./examples/cute/tutorial/[EXAMPLE_NAME]_sycl
```

## CUTLASS Example

>  Currently, the the example `14_amper_tf32_tensorop_gemm` has been implemented for SYCL on Nvidia Ampere architecture. You can build this from your build directory by running :

Reviewer 2: ```suggestion
 Currently, the example `14_amper_tf32_tensorop_gemm` has been implemented for SYCL on Nvidia Ampere architecture. You can build this from your build directory by running :
```
